tools/exporters/data_dictionary_builder.php

```diff
@@ -138,33 +140,34 @@
 $parameterCount =0;
 
 foreach ($instruments AS $instrument) {
-    $catId ="";
+    $catId = "";
+    $table = "";
     $items =explode("\n", trim($instrument));
     foreach ($items AS $item) {
         $paramId ="";
         $bits    =explode("{@}", trim($item));
         switch($bits[0]){
         case "table":
-            $table =$bits[1];
-            print "At $table\n";
+            $table = $bits[1];
+            print "Instrument: $table\n";
             break;
 
         case "title":
-            $title  = $bits[1];
-            $error  = $DB->insert(
+            $title = $bits[1];
+            $error = $DB->insert(
                 "parameter_type_category",
                 array(
-                 'Name' => $title,
-                 'Type' => 'Instrument',
+                 "Name" => $title,
+                 "Type" => "Instrument",
                 )
             );
-             $catId = $DB->lastInsertID;
-             $tblCount++;
+            $catId = $DB->lastInsertID;
+            $tblCount++;
             break;
 
         case "header":
             continue;
-            break;
+                break;
```

Comment:
Is `break` required after `continue`?

Reply:
good question from what I have seen it seems like it does not no. I will remove it

Reply:
no, both of them modify the control flow, the break will never be reachable.

Reply:
on second thought this might be a bit more difficult. was the intent of `continue; break;` to stop exit the switch statement only or the entire `foreach ($items AS $item) {` (line 146) ?? because if thats the case, we would need to replace continue break by `continue 2;`

Reply:
the break is never reachable. Removing it won't change any behaviour.